README.md

```diff
@@ -1,2 +1,38 @@
-# math-magicians
-### Math Magicians React Js Project.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+> This is my first project of the Math Magicians application buiding in React Js.
+
+
+## Run the Project Locally
+- To Clone the Project Locally Please click the following link.
+- [Math Magicians](https://github.com/Nizamuddin4493/math-magicians)
+- The click on Green Code button
+- Copy the SSH link and clone it Locally through your Code Editor's terminal.
+
```

Comment:
- [ ] Please it will be nice to update your `readme file` with explicit steps on how to clone and run your project locally as this will be of help to none technical users and some of them are potential recruiters. Something like so: 
`git clone https://github.com/Nizamuddin4493/math-magicians.git`
`cd math-magicians`
`npm install`
`npm start` .......